include/PrimeTower.h

```diff
@@ -40,7 +42,10 @@ class PrimeTower
     const unsigned int number_of_prime_tower_start_locations_ = 21; //!< The required size of \ref PrimeTower::wipe_locations
 
     MovesByExtruder prime_moves_; //!< For each extruder, the moves to be processed for actual priming.
+    std::map<size_t, std::map<size_t, Polygons>>
+        sparse_pattern_per_extruders_; //!< For each extruders combination, and for each actual extruder, the pattern to print on all layers where extruders are actually useless.
```

Comment:
From the variable name I would suspect the type to be `std::map<size_t, Polygons>`. I'm not sure from the comment why the type is `std::map<size_t, std::map<size_t, Polygons>>` instead. Can the comment be clarified a bit?

Reply:
I tried to do that already, but that was a difficult exercise :grimacing: will try something else